### PR TITLE
ci: add action to auto deploy using ansible

### PR DESCRIPTION
**Pull request by helder-moreira**

### CODE REVIEW

**Review by fadeev — APPROVED** [review]
Looks great! Thanks again @helder-moreira 👏

File: .ansible/roles/cosmos-faucet/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+faucet_version: master
+log_level: info
+key_name: faucet
+cli_name: gaiacli
```
Comment (ltacker) [inline .ansible/roles/cosmos-faucet/defaults/main.yml:4]: What gaia is used for here?

**Review by ltacker — APPROVED** [review]
💪

File: .ansible/roles/cosmos-faucet/defaults/main.yml (hunk 1 above)
Comment (helder-moreira) [reply .ansible/roles/cosmos-faucet/defaults/main.yml:4]: great catch! thank you! It should be `spnd`

Comment (helder-moreira) [reply .ansible/roles/cosmos-faucet/defaults/main.yml:4]: oh, actually those are the default values, as that role was initially designed for `gaia`. The value is overwritten [here](https://github.com/tendermint/spn/blob/helder/add-deploy-action/.ansible/playbook.yml#L58)

Comment (ltacker) [reply .ansible/roles/cosmos-faucet/defaults/main.yml:4]: Ok great 👍

**Pull request merged**